app/services/idv/profile_step.rb

```diff
@@ -63,7 +63,24 @@ def vendor_validator_class
 
     def update_idv_session
       idv_session.profile_confirmation = success
-      idv_session.resolution = vendor_validator.result
+      idv_session.resolution = proofer_resolution
+      idv_session.vendor_session_id = proofer_resolution.session_id
+      idv_session.normalized_applicant_params = normalized_applicant_to_hash
+      idv_session.resolution_successful = proofer_resolution.success?
+    end
+
+    def proofer_resolution
+      @resolution ||= vendor_validator.result
+    end
+
+    def normalized_applicant
+      @normalized_applicant ||= proofer_resolution.vendor_resp.normalized_applicant
+    end
+
+    def normalized_applicant_to_hash
+      normalized_applicant.instance_variables.each_with_object({}) do |var, hash|
+        hash[var.to_s.delete('@')] = normalized_applicant.instance_variable_get(var)
+      end
```

Comment:
Now that the other PR is merged, should we bump the gem and remove this? Or do we want to follow up with a separate PR?